ui/imports/utils/Utils.qml

```diff
@@ -853,10 +853,10 @@ QtObject {
                                        StatusQUtils.AmountsArithmetic.fromNumber(1, rawDecimals - gasDecimals))
     }
 
-    function getGasDecimalValue(chainID, gasValue, gasLimit) {
-        let rawValue = nativeTokenGasToRaw(chainID, gasValue)
-        rawValue = StatusQUtils.AmountsArithmetic.times(rawValue, StatusQUtils.AmountsArithmetic.fromNumber(1, gasLimit))
-        return nativeTokenRawToDecimal(chainID, rawValue)
+    function calculateGasCost(chainID, gasPriceInGwei, gasAmount) {
```

Comment:
This function is general, and I wouldn't put `Gwei` in a variable name. Maybe better to name it `gasPriceInGasUnit` or just `gasPrice` and add a comment.